yarn-project/prover-node/src/prover-node.ts

```diff
@@ -115,32 +118,40 @@ export class ProverNode implements ClaimsMonitorHandler, EpochMonitorHandler {
    */
   async handleEpochCompleted(epochNumber: bigint): Promise<void> {
     try {
+      // Construct a quote for the epoch
       const blocks = await this.l2BlockSource.getBlocksForEpoch(epochNumber);
       const partialQuote = await this.quoteProvider.getQuote(Number(epochNumber), blocks);
```

Comment:
In the original design, we had two env variables for default quote parameters.

i.e. PROVER_BASIS_POINT_FEE and PROVER_BOND_AMOUNT

Are we still doing this?

Reply:
Yep, those should be used by the quote provider to return the `partialQuote` here